Apps/phawswafv2/awswafv2_connector.py

```diff
@@ -200,15 +200,19 @@ def paginator(self, limit, action_result):
         if not self._create_client(action_result):
             return action_result.get_status()
 
-        dic_map = {
+        action_identifier_map = {
             'list_ip_sets': ['list_ip_sets', 'IPSets'],
             'list_acls': ['list_web_acls', 'WebACLs'],
             'add_ip': ['list_ip_sets', 'IPSets'],
             'delete_ip': ['list_ip_sets', 'IPSets']
         }
 
-        method_name = dic_map.get(self.get_action_identifier())[0]
-        set_name = dic_map.get(self.get_action_identifier())[1]
+        action_identifier = self.get_action_identifier()
+        if action_identifier not in action_identifier_map.keys():
+            return {}
```

Comment:
should this be a list to match the return type of `set_list` at the end?

Reply:
yes, return type should be a list. Corrected. Thanks.!